models/comments.model.js

```diff
@@ -23,3 +23,15 @@ exports.insertCommentToArticle = (article_id, body) => {
       return rows[0];
     });
 };
+
+exports.deleteCommentByCommentID = (comment_id) => {
+  return db
+    .query(`DELETE FROM comments WHERE comment_id=$1 returning *`, [comment_id])
+    .then(({ rows: deleted_comments }) => {
+      if (deleted_comments.length === 1) {
+        return Promise.resolve();
```

Comment:
because we dont need anything back from this I would probably just have if length ===0 Promise reject :)

Reply:
changed :)